src/modules/75-cd/components/PipelineSteps/TerraformPlan/TerraformPlan.tsx

```diff
@@ -102,17 +127,148 @@ function TerraformPlanWidget(
+              // Handle on success
+            }}
+            isEditMode={isEditMode}
+            setIsEditMode={setIsEditMode}
+            connectorInfo={undefined}
+            accountId={accountId}
+            orgIdentifier={orgIdentifier}
+            projectIdentifier={projectIdentifier}
+          />
+        ) : null}
+        {connectorType === Connectors.GITHUB ? (
+          <StepGithubAuthentication
+            name={getString('credentials')}
+            onConnectorCreated={() => {
+              // Handle on success
+            }}
+            isEditMode={isEditMode}
+            setIsEditMode={setIsEditMode}
+            connectorInfo={undefined}
+            accountId={accountId}
+            orgIdentifier={orgIdentifier}
+            projectIdentifier={projectIdentifier}
+          />
+        ) : null}
+        {connectorType === Connectors.BITBUCKET ? (
+          <StepBitbucketAuthentication
+            name={getString('credentials')}
+            onConnectorCreated={() => {
+              // Handle on success
+            }}
+            isEditMode={isEditMode}
+            setIsEditMode={setIsEditMode}
+            connectorInfo={undefined}
+            accountId={accountId}
+            orgIdentifier={orgIdentifier}
+            projectIdentifier={projectIdentifier}
+          />
+        ) : null}
+        {connectorType === Connectors.GITLAB ? (
+          <StepGitlabAuthentication
+            name={getString('credentials')}
+            identifier={CONNECTOR_CREDENTIALS_STEP_IDENTIFIER}
+            onConnectorCreated={() => {
+              // Handle on success
+            }}
+            isEditMode={isEditMode}
+            setIsEditMode={setIsEditMode}
+            connectorInfo={undefined}
+            accountId={accountId}
+            orgIdentifier={orgIdentifier}
+            projectIdentifier={projectIdentifier}
+          />
+        ) : null}
+        {connectorType === Connectors.ARTIFACTORY ? (
+          <StepArtifactoryAuthentication
+            name={getString('details')}
+            identifier={CONNECTOR_CREDENTIALS_STEP_IDENTIFIER}
+            isEditMode={isEditMode}
+            setIsEditMode={setIsEditMode}
+            connectorInfo={undefined}
+            accountId={accountId}
+            orgIdentifier={orgIdentifier}
+            projectIdentifier={projectIdentifier}
+          />
+        ) : null}
+        <DelegateSelectorStep
+          name={getString('delegate.DelegateselectionLabel')}
+          isEditMode={isEditMode}
+          setIsEditMode={setIsEditMode}
+          buildPayload={buildPayload}
+          connectorInfo={undefined}
+        />
+        <VerifyOutOfClusterDelegate
+          name={getString('connectors.stepThreeName')}
+          connectorInfo={undefined}
+          isStep={true}
+          isLastStep={false}
+          type={connectorType}
+        />
+      </StepWizard>
+    )
+  }
+
+  const onStepChange = (arg: StepChangeData<any>): void => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232248)
```suggestion
  const onStepChange = (arg: StepChangeData<unknown>): void => {
```